PROCESSES.md

```diff
@@ -0,0 +1,11 @@
+# Processes
+
+This document outlines processes and procedures for some common tasks in the charts repository.
+
+## Adding chart maintainers
```

Comment:
Perhaps it would help to also add sections for responsibilities of chart maintainers and repo admin roles? Per chat and your note about it as well here? https://github.com/prometheus-community/helm-charts/issues/38#issuecomment-687651538

Reply:
That's a good idea. I would love to split it up into smaller PRs as it's easier to review and get it merged.